scripts/utils/generate_ts.sh

```diff
@@ -112,15 +111,15 @@ EOF
       do
         ts_name=$(basename "$f")
         if [ -f "addon_ts/${ts_name}" ]; then
-          printn Y "File ${ts_name} exists, updating with branch strings..."
+          print Y "File ${ts_name} exists, updating with branch strings..."
           ${QT_HOST_BINS}/lconvert -i "addon_ts/${ts_name}" "$f" -o tmp.ts || die
           mv tmp.ts "addon_ts/${ts_name}"
         else
-          printn Y "File ${ts_name} does not exist, copying over..."
+          print Y "File ${ts_name} does not exist, copying over..."
           cp "$f" addon_ts/
         fi
+        rm $f || die
       done
-      rm addons/generated/addons/*.ts || die
```

Comment:
Why?

Reply:
On the `releases/2.16.0` branches, the addons are built with cmake, which puts the translation files in a different location, and this globbing pattern doesn't match any files. This causes `rm` to terminate with an error.